src/app/sign-in/sign-in.component.scss

```diff
@@ -0,0 +1,56 @@
+.sign-in-page-background {
+  height: 100%;
+  background: url("../../assets/auth-background.jpg") no-repeat;
+  background-size: cover;
+}
+
+.container {
+  max-width: 100rem;
+  padding: 0 1rem;
+  height: 100%;
+  margin-inline: auto;
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.sign-in-form {
+  max-width: 32rem;
+  width: 100%;
+
+  &__title {
+    font-family: 'impact', sans-serif;
+    font-size: 2.625rem;
+    color: #F8BB15;
+    margin-bottom: 2.5rem;
+  }
+
+  @media (max-width: 400px) {
+    &__title {
+      font-size: 2.25rem;
+      text-align: center;
+    }
+  }
+
+  &__form {
+    display: flex;
+    flex-flow: column;
+
+    .form__select-trigger {
+      color: white !important;
+    }
+  }
+
+  &__confirm-button {
+    width: 100%;
+    background: #F8BB15 !important;
```

Comment:
i guess you should modify material theme instead of overriding styles using !important

Reply:
fixed